doc/dev/GETTING_STARTED.md

```diff
@@ -0,0 +1,162 @@
+# Getting started guide
+
+This is a getting started guide, aimed to people that are not used
+to C++ programming, configuring and compiling. If you are already used
+to such processes, you may be looking for our build guide.
+
+This guide will help you to compile your own version of F3D with required dependencies.
+Optional dependencies or covered in the build guide.
+
+ - If you are using [Linux]
+ - If you are using [Windows]
+ - If you are using [macOS]
+
+## Linux
+
+### Install dependencies
+
+First make sure your system is up-to-date,
+then install the required dependencies using the terminal.
+
+#### Debian/Ubuntu/Mint
+
+```
+sudo apt install build-essential git git-lfs cmake libvtk9-dev
```

Comment:
how about mentioning python3  also I feel its good to mention the recommended Ubuntu version for people who use ubuntu

Reply:
It is out of place to suggest an OS, but I can mention on which OS this was tested

Python3 is not a required dependency at all.